### PR TITLE
implemented pagination and sorting of output data, added swagger

### PR DESCRIPTION
**Pull request by DmytroV95**

### CODE REVIEW

**Review by irynamekh — APPROVED** [review]
Good job!

**Review by vovasalyha — CHANGES_REQUESTED** [review]
Solution of the task is good, but resolve merge conflicts (I suspect it's because of added validation starter which should be added in the previous task)

File: pom.xml [hunk 1]
```diff
@@ -41,6 +41,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-validation</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+            <version>2.1.0</version>
```
Comment (nacenik) [inline pom.xml:47]: i'd recommend you put this version inside properties

**Review by nacenik — APPROVED** [review]
LGTM! 👍

**Pull request merged**